src/test/util_sds_check/util_sds_check.h

```diff
@@ -0,0 +1,58 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright 2025, Hewlett Packard Enterprise Development LP */
+
+/*
+ * util_sds_check.h -- header for unit test
+ */
+
+#include <stdlib.h>
+#include <stdbool.h>
+#include "unittest.h"
+#include "shutdown_state.h"
+#include "set.h"
+
+int os_part_deep_common(struct pool_replica *rep, unsigned partidx, void *addr,
+	size_t len, int flush);
+/* This function does not take part in this unit test. */
+int
+os_part_deep_common(struct pool_replica *rep, unsigned partidx, void *addr,
+	size_t len, int flush)
+{
+	return 0;
+}
+
+#define DIRTY_SET 1
+#define DIRTY_CLEAR 0
+
+/* Invalid value for all test cases */
+#define INVALID_VALUE 0
+
+#define CHECKSUM 0
+#define UUID 1
+#define USC 1
```

Comment:
```suggestion
#define INVALID_VALUE 0x60bab3f082b07f57
#define UUID 0xbe4919b3edce624a
#define USC 0x14796ef67d593c7b
```

Please note these values are 64 bits long. It is a lot. Using as much of the available memory as possible you make testing more robust. 💪 Here are my examples for the values you can use.

Reply:
Done.